solvis/filter/rupture_id_filter.py

```diff
@@ -300,12 +306,12 @@ def for_polygon(
         Returns:
             A chainable set of rupture_ids matching the filter arguments.
         """
-        df0 = gpd.GeoDataFrame(self._solution.solution_file.fault_sections)
+        df0: pandera.typing.pandas.DataFrame[Any] = gpd.GeoDataFrame(self._solution.solution_file.fault_sections)
```

Comment:
Question for my own education only: why is `pandera.typing.pandas.DataFrame` used to type a `GeoDataFrame` but `pandas.DataFrame` used to type a regular dataframe?

Reply:
Actually both are probably fine. I just chose something to resolve mypy typing complaints, bu ton reflection I'll try  other as it's the more obvious type to use. Thanks @chrisdicaprio

Reply:
making the change for #74 @chrisdicaprio